### DISCUSSION TITLE
RaspAP problem

### DISCUSSION THREAD

**Discussion by MrDLoomis** [RaspAP/raspap-webgui#1187] [General]
Hi,

Please be gentle.  This is my first time posting here, so this may be in the wrong place.  

I've searched high and low in the forums and can't find an answer.  I'm fairly savvy with Linux and Raspberry Pi's.  

I did a clean install of RaspAP.  After reboot, I  can SSH into the pi.  I'm using the on-board wifi as wlan0 and a USB wifi adapter as wlan1.  I can connect to the raspi-webgui at 10.1.141.1 using my iphone.  I can see the webgui there.  I can SSH to the pi via wlan1 which is connected to my home network at a 192.168.xxx.xxx address.  I can also access the webgui through wlan1 as well.  

Nothing I do seems to make the hospot (wlan0) talk to wlan1 and let me access the internet or browse.  Each seem to be working fine on their own, but not together.  The point of the AP is to be able to access the internet, which isn't happening.  I have my phone connected to the webgui network, it gets an IP.  Shouldn't the AP connect the two?  

Also, I have the wpa_supplicant.conf in /etc/wpa_supplicant/ configured for my network.  Do I have to change that every time I go to another network?  My plan was to be able to travel with the RaspAP and use it to attach multiple devices to hotel wifi.  I would think the Wireless Client scan should bring up the local wifi options.  

Thanks!

**Comment by billz** [comment]
Configuring an AP with two wireless adapters has a [dedicated FAQ](https://docs.raspap.com/faq/#interfaces).

RaspAP does not automatically set interface metrics for you, but you can easily do this via the UI.

Related discussion #1171 and [Reddit thread](https://www.reddit.com/r/RaspAP/comments/vl1g53/why_would_i_have_to_change_metrics/).

**Reply by MrDLoomis** [reply]
Thank you!  Changing the metric as suggested in the related thread fixed the problem.  I really appreciate your help!

One more question:  I'd like to be able to travel with it and use it to access hotel wifi's.  Do I have to manually change the wpa_supplicant file each time?

**Reply by billz** [reply]
RaspAP's built-in WiFi Client configuration provides a UI to modify your wpa_supplicant.
